### PR TITLE
Files needed for navbar to work

### PR DESCRIPTION
**Pull request by Rguerrero19**
completed the operation of the navbar task with its responsive
Captura de pantalla de 2023-08-15 19-53-54
Captura de pantalla de 2023-08-15 19-54-06

### CODE REVIEW

File: css-index.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@media (min-width:900px){
+  .container{
+    width: auto;
+    border: 3px solid black;
```
Comment (jesusMava) [inline css-index.css:10]: pregunta, por un border negro si tu imagen de background es del mismo color? jeje

File: css-index.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    display: grid;
+    grid-template-columns: repeat(4,1fr);
+    grid-auto-rows: 100px;
+  }
+  .container > div{
```
Comment (jesusMava) [inline css-index.css]: good job man, I really glad to see that u have started to use css nomenclature

File: css-index.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    border: 1px solid black;
+  }
+  .navegation{
+    display: flex;
+    align-self: flex-end;
+    grid-column: 1/5;
+  }
+  .name{
+    display: flex;
```
Comment (jesusMava) [inline css-index.css:29]: this is only a div, why should have the property flex? there is not anything inside 🤔

File: css-index.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    width: auto;
+    border: 3px solid black;
+    display: grid;
+    grid-template-columns: repeat(3,1fr);
+    grid-auto-rows: 100px;
+  }
+  .container > div{
+    border: 1px solid white;
+  }
+  .navegation{
```
Comment (jesusMava) [inline css-index.css]: there is some style that you repeat, why dont you move to the base style, without a media query hehe

Comment (jesusMava) [inline css-index.css]: there is some style that you repeat, why dont you move to the base style, without a media query hehe

File: css-index.css [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    color: aliceblue;
+    background-color: black;
+    border-radius: 30px;
+    display:flex;
+    padding-bottom: 10px;
+    justify-content: flex-end;
+  }
+  
+}
+@media (max-width:745px){
```
Comment (jesusMava) [inline css-index.css]: good job, you have used the medias correctly, nice :D

**Review by jesusMava — CHANGES_REQUESTED** [review]

**Comment by jesusMava** [comment]
this is what I see...

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  <head>
+    <title></title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css"href="css-index.css">
+  </head>
+  <body background="pexels-aleksandar-pasaric-2603464.jpg">
+    <div class="container">
+      <nav class="navegation">
+        <div class="name">Hello Iam RamonMg</div>
```
Comment (jesusMava) [inline index.html]: Hello I am RamonMG

File: index.html [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+        <div class="buttons">
+          <button>Formacion</button>
```
Comment (jesusMava) [inline index.html]: Training or Education instead of Formacion

File: css-index.css (hunk 1 above)
Comment (Rguerrero19) [reply css-index.css:10]: aaaa sorry I forgot to remove that it was only to distinguish that it was working correctly

Comment (jesusMava) [reply css-index.css:10]: aa dont worry, I understand you

**Comment by jesusMava** [comment]
Wait a moment why are you sending me the form if we are working with the nav and you are not changing things that I mentioned above? 🤔

File: css-index.css (hunk 1 above)
Comment (Rguerrero19) [reply css-index.css:10]: At the moment I occupy them to limit the spaces of my document

File: css-index.css (hunk 4 above)
Comment (Rguerrero19) [reply css-index.css]: It is necessary to delimit the navbar area

Comment (Rguerrero19) [reply css-index.css]: I use them as a guide to see how my divs are being limited

File: css-index.css (hunk 5 above)
Comment (Rguerrero19) [reply css-index.css]: thank you

File: index.html (hunk 7 above)
Comment (Rguerrero19) [reply index.html]: agreed corrected

**Review by jesusMava — APPROVED** [review]

**Pull request merged**